### DISCUSSION TITLE
Image upload in Supabase and React

### DISCUSSION THREAD

**Discussion by OmaJuliet** [supabase/supabase#12301] [Questions]
Can anyone help me out on how I can enable image upload in Supabase for my React CRUD app?

I have a table with name, email and other details and it works just fine for creating, updating and deleting but I can't seem to figure out how to do the same for images

**Comment by chkmls** [comment]
Did you figured it out?

**Reply by jopfre** [reply]
Images need to be stored in "storage"

https://supabase.com/docs/reference/javascript/storage-from-upload
